resources/views/hr/addition/index.blade.php

```diff
@@ -248,7 +248,7 @@
 			<div class="modal-body">Are you sure to delete this record?</div>
 			<div class="modal-footer">
 				<button type="button" class="btn btn-default" onClick="dismissModel()">Cancel</button>
-				<form id="delete-frm" class="" action="" method="POST">
+				<form id="delete-frm" class="" action="{{route('additions.addition.destroy', $addition->id )}}" method="POST">
```

Comment:
This didn't break at yours @ScriptHubs ? haha

There's a working javascript function to take care of the `action` attribute ;) The same also goes to `deduction`, `loan`, and `overtime`. So I reverted these changes to fix the problem.